examples/transform_speed.ipynb

```diff
@@ -0,0 +1,365 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# prepare list of image names and segmentation names\n",
+    "images = sorted(glob(os.path.join(tempdir,'im*.nii.gz')))\n",
+    "segs = sorted(glob(os.path.join(tempdir,'seg*.nii.gz')))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Test image loading with minimal preprocessing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "torch.Size([3, 1, 256, 256, 256]) torch.Size([3, 1, 256, 256, 256])\n"
+     ]
+    }
+   ],
+   "source": [
+    "imtrans = Compose([\n",
+    "    AddChannel(),\n",
+    "    ToTensor()\n",
+    "])    \n",
+    "\n",
+    "segtrans = Compose([\n",
+    "    AddChannel(),\n",
+    "    ToTensor()\n",
+    "])    \n",
+    "    \n",
+    "ds = NiftiDataset(images, segs, transform=imtrans, seg_transform=segtrans)\n",
+    "loader = DataLoader(ds, batch_size=3, num_workers=8)\n",
+    "\n",
+    "im, seg = monai.utils.misc.first(loader)\n",
+    "print(im.shape, seg.shape)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "3.51 s ± 119 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)\n"
+     ]
+    }
+   ],
+   "source": [
+    "%timeit data = next(iter(loader))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Test image-patch loading with CPU multi-processing:\n",
+    "\n",
+    "- rotate (256, 256, 256)-voxel in the plane axes=(1, 2)\n",
+    "- extract random (64, 64, 64) patches\n",
+    "- implemented in MONAI implemented using ` scipy.ndimage.rotate`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "torch.Size([3, 1, 64, 64, 64]) torch.Size([3, 1, 64, 64, 64])\n"
+     ]
+    }
+   ],
+   "source": [
+    "images = sorted(glob(os.path.join(tempdir,'im*.nii.gz')))\n",
+    "segs = sorted(glob(os.path.join(tempdir,'seg*.nii.gz')))\n",
+    "\n",
+    "imtrans = Compose([\n",
+    "    Rescale(),\n",
+    "    AddChannel(),\n",
+    "    Rotate(angle=45.),\n",
+    "    UniformRandomPatch((64, 64, 64)),\n",
+    "    ToTensor()\n",
+    "])    \n",
+    "\n",
+    "segtrans = Compose([\n",
+    "    AddChannel(),\n",
+    "    Rotate(angle=45.),\n",
+    "    UniformRandomPatch((64, 64, 64)),\n",
+    "    ToTensor()\n",
+    "])    \n",
+    "    \n",
+    "ds = NiftiDataset(images, segs, transform=imtrans, seg_transform=segtrans)\n",
+    "loader = DataLoader(ds, batch_size=3, num_workers=4, pin_memory=torch.cuda.is_available())\n",
```

Comment:
As num_workers obviously affect CPU performance and seed, is this "num_workers=4" the best value on your CPU?
And maybe you need to state your CPU type in the markdown introduction.
Thanks.